events/wc_druidic_events.txt

```diff
@@ -105,7 +143,7 @@ character_event = {
 			}
 		}
 		
-		event_target:target_mage = { narrative_event = { id = WCDRU.300 } }
```

Comment:
Hmm? Why it's changed?

Reply:
A leftover from when there were 2 event chains (298 to 300, 299 to 301). Fixed.